tests/tests/functional/chart_test.py

```diff
@@ -105,6 +107,18 @@ def run_verifier(image, profile_type, chart_location):
 
     return output.decode("utf-8")
 
+def run_tarball_image(tarball_name,profile_type, chart_location):
+    print(f"Run tarball image from {tarball_name}")
+
+    tar = tarfile.open(tarball_name, "r:gz")
+
+    tar.extractall(path="./test_verifier")
+
+    out = subprocess.run(["./test_verifier/chart-verifier","verify","--set",f"profile.vendorType={profile_type}",chart_location],capture_output=True)
```

Comment:
So much simpler the tar test.

Reply:
Curious... how come this works without PR 221 merged. In the Docker Images we already have, helm, oc and kubectl binaries, but don't see here where they get added.

Reply:
In the test I see is not getting this far as it fails when tart is being opened.

Reply:
The test fails because it uses the build.yaml file from the main branch which does not include the changes to create the tarball. I did include a link to a PR which uses the new build.yaml. Guess the tarball test passes because helm and oc are installed.

Reply:
Got it.